pkg/provider/provider.go

```diff
@@ -170,15 +174,15 @@ func (p *wavefrontProvider) getMultiple(info provider.CustomMetricInfo, namespac
 	return p.metricsFor(queryResult, namespace, info, resourceNames)
 }
 
```

Comment:
From https://github.com/kubernetes-sigs/custom-metrics-apiserver/blob/8c12d6d923621e72a9d3fef902e48b83d1bbfe4b/pkg/provider/interfaces.go#L80

>// For queries that use label selectors, it is up to the
// implementor to decide how to make use of the label selector --
// they may wish to query the main Kubernetes API server, or may
// wish to simply make use of stored information in their TSDB.